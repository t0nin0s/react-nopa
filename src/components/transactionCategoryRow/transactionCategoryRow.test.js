import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import TransactionCategoryRow from './TransactionCategoryRow'

test('TransactionCategoryRow component renders as expected', () => {
  const component = shallow(<TransactionCategoryRow />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
