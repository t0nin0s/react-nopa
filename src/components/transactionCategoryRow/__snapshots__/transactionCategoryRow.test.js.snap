// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionCategoryRow component renders as expected 1`] = `
<div
  className="categoryRow"
/>
`;
